modules/issue_tracker/jsx/columnFormatter.js

```diff
@@ -26,7 +26,7 @@ function formatColumn(column, cell, rowData, rowHeaders) {
   if (column === 'Title') {
     let cellLinks = [];
     cellLinks.push(
-      <a href={loris.BaseURL + '/issue_tracker/issue/' + row['Issue ID']}>
+      <a href={loris.BaseURL + '/issue_tracker/issue/?issueID=' + row['Issue ID']}>
```

Comment:
This is not correct. The ID should be coming from the handle method parsing the URL at https://github.com/aces/Loris/blob/minor/modules/issue_tracker/php/issue.class.inc#L144

Reply:
@driusan If use RESTful API in this case, the "QueryString.get()" will get a null value. 
Major branch is also using "?issueID=" on URL. I think if you need this to be fixed, it should on Major branch.

Reply:
in that case the major branch is wrong and also needs to be fixed. The ID should be coming from the URL, not the query string

Reply:
@driusan will look into the `handle` function and try to solve the issue

Reply:
@driusan @ridz1208 could u review again?